### ISSUE TITLE
[Feb 2025] Accessibility Audit: Header and user menu drop-down and touchpoints modal

### ISSUE THREAD

**Issue by DanielSass** [CDCgov/prime-simplereport#8511]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the header and user menu dropdown. Do the page scan as well as any relevant guided tests.
Use cases to test:
* facility select
* org with 1 facility, and multiple facilities
* Add new facility form (when org admin user logs in to org with no facility created)
* view header and user menu as someone with test_only access, standard_user access, and admin access
* touchpoints button + modal

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

### Additional context
See tickets from previous audits #5779, #6335

**Comment by emyl3** [comment]
Findings
- Facility select [dev.simplereport.gov-Select-facility-2025-03-04.json](https://github.com/user-attachments/files/19074263/dev.simplereport.gov-Select-facility-2025-03-04.json)

**Comment by emyl3** [comment]
Issues created:
- https://github.com/CDCgov/prime-simplereport/issues/8573

**Comment by emyl3** [comment]
Ready for review!
